android/expoview/src/main/java/host/exp/exponent/experience/ReactNativeActivity.java

```diff
@@ -224,6 +229,12 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
       mReactInstanceManager.call("showDevOptionsDialog");
       return true;
     }
+    boolean didDoubleTapR = Assertions.assertNotNull(mDoubleTapReloadRecognizer)
+        .didDoubleTapR(keyCode, getCurrentFocus());
+    if (didDoubleTapR) {
+      ((DevSupportManager) mReactInstanceManager.call("getDevSupportManager")).handleReloadJS();
```

Comment:
we need to check `mReactInstanceManager != null && mReactInstanceManager.isNotNull()` like we do above. also need to check that getDevSupportManager isn't null since we could not be in dev mode